.github/ISSUE_TEMPLATE/cfp.md

```diff
@@ -0,0 +1,81 @@
+name: 💬 Call for Proposal
+description: Propose a session for a Schrödinger Session
+body:
+  - type: markdown
+    attributes:
+      value: |
+        # Guidelines
+        
+        Remember to respect the following things while presenting your talk
+
+        - Respect the Code of Conduct
+        - Use an inclusive language
+  - type: dropdown
+    id: format
+    attributes:
+      label: Format
+      description: Please select the format of your talk
+      options:
+        - Talk (30 minutes)
+        - Lightning Talk (15 minutes)
+        - Workshop (> 2 hours)
+    validations:
+      required: true
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      placeholder: Describe your Session
+    validations:
+      required: true
+  - type: input
+    id: slides
+    attributes:
+      label: Link to the slides
+      description: The slides of your session
+    validations:
+      required: false
+  - id: places
+    type: checkboxes
+    attributes:
+      label: In which city you may available to travel in to perform a session?
+      options:
+        - label: Florence, Italy 🇮🇹
+          required: false
+        - label: Verona, Italy 🇮🇹
+          required: false
+        - label: Paris, France 🇫🇷
+          required: false
```

Comment:
Let's add Brussels and Torino and Milan as well, as we may have a partnership with talent garden